src/datetime/YearsView.js

```diff
@@ -32,16 +31,15 @@ export default class YearsView extends React.Component {
 		);
 	}
 
-	renderYears( viewYear ) {
+	renderYears() {
+		const viewYear = this.getViewYear();
 		// 12 years in 3 rows for every view
 		let rows = [ [], [], [] ];
-		let selectedYear = this.props.selectedDate && this.props.selectedDate.year();
-
 		for ( let year = viewYear - 1; year < viewYear + 11; year++ ) {
 			let row = this.getRow( rows, year - viewYear );
```

Comment:
We can remove `this.` here (this.getRow)

Reply:
Thanks, removed!